mailbox/backup/src/test/java/org/apache/james/mailbox/backup/ZipperTest.java

```diff
@@ -196,6 +201,26 @@ void archiveShouldWriteMailboxAnnotationsMetadataWhenTwoPresent() throws Excepti
         }
     }
 
+    private MailboxAnnotation getAnnotationForIndex(int i) {
+      return MailboxAnnotation.newInstance(new MailboxAnnotationKey("/annotation" + i + "/test"), "annotation " + i + " content");
+    }
+
+    private final List<MailboxAnnotation> oneThousandAnnotations =   ContiguousSet.create(Range.closed(0, 999), DiscreteDomain.integers()).stream()
```

Comment:
`oneThousandAnnotations =   ContiguousSet`

Multiple spaces spotted.

Reply:
`ContiguousSet.create(Range.closed(0, 999), DiscreteDomain.integers()).stream()`

We tend to use the more simple

`IntStream.range(0, 999)`